aperturedb/ConnectionLoader.py

```diff
@@ -76,14 +85,19 @@ def validate(self):
 class ConnectionLoader(ParallelLoader.ParallelLoader):
     """**ApertureDB Connection Loader.**
 
-        This class is to be used in combination with a "generator".
-        The generator must be an iterable object that generated "entity_data"
-        elements.
+    This executes in conjunction with a **generator** object for example :class:`~aperturedb.ConnectionLoader.ConnectionGeneratorCSV`,
+    which is a class that implements iterable inteface and generates "connection data" elements.
 
     Example::
```

Comment:
```suggestion
    Example:
```

Reply:
That's a special directive for the auto api .

Reply:
oh, my bad.